pkg/sql2pgroll/expect/create_table.go

```diff
@@ -129,3 +129,88 @@ var CreateTableOp11 = &migrations.OpCreateTable{
 		},
 	},
 }
+
+var CreateTableOp12 = &migrations.OpCreateTable{
```

Comment:
Since all of these only differ based on the `OnDelete` action maybe a helper method would be useful?

Reply:
yes, will do this separately 👍